### PR TITLE
Create scenario to split bill

### PR DESCRIPTION
**Pull request by Mdaiki0730**
## Description
This is a webhook implementation for the bill splitter.
It will allow you to get a response like the one in the image.

### CODE REVIEW

**Review by ulbqb — APPROVED** [review]
LGTM

File: bot/index.ts [hunk 1]
```diff
@@ -45,13 +66,193 @@ async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
...
+              {
+                type: "text",
+                text: "RECEIPT",
+                weight: "bold",
+                color: "#1DB446",
+                size: "sm"
+              },
+              {
+                type: "text",
+                text: "Carol's Birthday Party",
```
Comment (ulbqb) [inline bot/index.ts:91]:
```suggestion
                text: "Kaia's Birthday Party",
```
How about Kaia's birthday party? Kaia mainnet launches today!

File: bot/index.ts [hunk 2]
```diff
@@ -19,24 +17,47 @@ const bot = createKaiaBotClient({
...
 
-async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
+async function commandNotFound(bot: KaiaBotClient, event: MessageEvent) {
   try {
     const to = event.source.userId || "";
     const messages: Array<TextMessage> = [
       {
         type: "text",
-        text: "Hello world",
+        text: "Hello. Let's start making payment",
```
Comment (ulbqb) [inline bot/index.ts:40]:
```suggestion
        text: "Hello. Let's start splitting bill.",
```

action: 
- Check receipt
- Pay to payer
- Add order

File: bot/index.ts [hunk 3]
```diff
@@ -45,13 +66,193 @@ async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
...
+              }
+            ]
+          },
+          styles: {
+            footer: {
+              separator: true
+            }
+          }
+        },
+        quickReply: {
```
Comment (ulbqb) [inline bot/index.ts:217]: action:

- Pay to payer
- Add receipt

File: bot/index.ts [hunk 4]
```diff
@@ -45,13 +66,193 @@ async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
...
+                    contents: [
+                      {
+                        type: "text",
+                        text: "NUMBER OF PEOPLE",
+                        size: "xs",
+                        color: "#555555"
+                      },
+                      {
+                        type: "text",
+                        text: "5",
```
Comment (ulbqb) [inline bot/index.ts:181]: If you adopt Kaia's party, 5 people is too few. How about 500?

File: bot/index.ts [hunk 5]
```diff
@@ -45,13 +66,193 @@ async function say_hello(bot: KaiaBotClient, event: MessageEvent) {
...
+                        size: "sm",
+                        color: "#111111",
+                        align: "end"
+                      }
+                    ]
+                  },
+                  {
+                    type: "box",
+                    layout: "horizontal",
+                    contents: [
```
Comment (ulbqb) [inline bot/index.ts:129]: If you adopt Kaia's party, it's good to order drinks.

File: bot/index.ts (hunk 1 above)
Comment (Mdaiki0730) [reply bot/index.ts:91]: That's good (:

File: bot/index.ts (hunk 4 above)
Comment (Mdaiki0730) [reply bot/index.ts:181]: If there are too many, it will be difficult to visualize, so let's make it 50.

**Pull request merged**